transpile.js

```diff
@@ -880,9 +916,10 @@ if (classes === null) {
 }
 
 // HINT: if we're going to support specific class definitions this process won't work anymore as it will override the definitions.
-exportTypeScriptDeclarations (classes)
+exportTypeScriptDeclarations (classes)  // we use typescript?
```

Comment:
> we use typescript?

We do, however, our typescript support is currently very rudimentary. It was added for compatibility with the GDAX TT and for those who have requested it. We are going to enhance it further, and will probably make a transition from a loosely-typed to a strictly-typed language in order to actually support more languages, including Rust, Go, Typescript and maybe even C/C++.

Reply:
Wow that's amazing, I'm going to study computer science at Berekeley next year and would love to help you with your transpiling endevours. The typescript will evolve into a strange meta-language (the javascript already has kind of) since we will only be able to support a subsect of its syntax.

Reply:
For some reason, typescript people prefer it all in one file (this is the way the code is structured conventionally): https://github.com/ccxt/ccxt/blob/master/ccxt.d.ts

I think this is because they don't have to bother with imports and include paths (means more portability).